index.html

```diff
@@ -12,9 +12,49 @@
   <body class="p-0 p-sm-1 p-sm-1 p-md-2 p-lg-3">
     <header class="header container bg-primary p-3">
       <h1 class="h1 text-center text-light fs-1">Awsome Books</h1>
+
+      <!-- -------------- -->
+      <!-- Navigation Bar -->
+      <!-- -------------- -->
+      <div class="container">
+        <div
+          class="row row-cols-1 g-2 row-cols-md-4 bg-danger-subtle align-items-center p-3"
+        >
+          <div class="col-12 col-md-6">
+            <p class="mb-0 fs-4 align-self-center">Awsome Book</p>
+          </div>
+          <div class="col-12 col-md-2">
+            <button
+              onclick="showSection(event, 'booklist')"
+              class="btn btn-light w-100 btn--nav"
+            >
+              List
+            </button>
+          </div>
+          <div class="col-12 col-md-2">
+            <button
+              onclick="showSection(event, 'addbook')"
+              class="btn btn-light w-100 btn--nav"
+            >
```

Comment:
- [ ] Please kindly design this section according to the wireframe, this will show that we pay attention to details. 👍

## Wireframe
<img width="218" alt="Untitled" src="https://github.com/M-Anwar-Hussaini/Awesome-Library/assets/103386183/5be70a2a-6e1e-4ba7-9823-1f9b817f118d">

- [optional] I got the error below when I tried to add a book to the list. Kindly fix this to make your project complete. **Note: It's optional because it's not part of the requirement for this milestone but I strongly implore you to implement it. Thanks.** 🙏 

 
<img width="930" alt="Untitled" src="https://github.com/M-Anwar-Hussaini/Awesome-Library/assets/103386183/679c2aea-d416-4b4e-8a2d-22cf0299ad97">